priv/repo/migrations/20170419082821_create_log_changes_table.exs

```diff
@@ -3,7 +3,14 @@ defmodule EctoTrail.TestRepo.Migrations.CreateAuditLogTable do
   use Ecto.Migration
 
   def change do
-    create table(:audit_log, primary_key: false) do
+    :ecto_trail
```

Comment:
@AndrewDryga is this way your idea?

Reply:
It's more that we don't need `change/0`. Instead, it's better to have `change/1` where `table_name` would default to `:audit_log`. 

By doing that you can customize table name whenever you call `change/1` from the migration. Migration itself should be probably moved to `lib`. I imagine application to simply call this change hook, rather than re-using package-provided migration.